### DISCUSSION TITLE
Setup and build instruction for local development

### DISCUSSION THREAD

**Discussion by hieuwu** [supabase-community/supabase-kt#274] [Q&A] [answered]
Hi @jan-tennert  , do we have any guidelines for building and testing locally with `supabase-kt`? It would be nice if we have an instruction page for setting up the whole project or a separate module. It will be easier for contributor to test their changes

**Comment by jan-tennert** [comment, marked as answer]
Hi, @hieuwu we don't have currently, but it's a good suggestion. The module "test" (which gets included in settings.gradle.kts, but doesn't actually exist on the repo as it's in the gitignore) can be created locally and used. I'd have this module on the repo as a template for testing, but I don't think I can control that contributers accidentally modify this module and then forget to remove their changes on commit etc. Ideas are welcome!
